src/cholesky.jl

```diff
@@ -7,6 +7,8 @@ end
 @inline function Base.chol(A::Base.LinAlg.RealHermSymComplexHerm{<:Real, <:StaticMatrix})
     _chol(Size(A), A.data)
 end
+@inline Base.LinAlg._chol!(A::StaticMatrix, ::Type{UpperTriangular}) = chol(A)
```

Comment:
Convention in Base for this functions is that the `!` indicates that `_chol!` has the licence to modify, but does not need to. Compare https://github.com/JuliaLang/julia/blob/9dbaed2fcfd0cc209e4c4a3bad231361b6aa29be/base/linalg/cholesky.jl#L118